include/avif/avif.h

```diff
@@ -94,7 +94,7 @@ typedef enum avifPlanesFlag
 } avifPlanesFlag;
 typedef uint32_t avifPlanesFlags;
 
-enum avifChannelIndex
+typedef enum avifChannelIndex
```

Comment:
- I removed R,G,B,A from `avifChannelIndex`.
- I added the generic `avifChannel` enum in `aviftest_helpers.h`. For now it contains just R,G,B,A but it can be extended to Y,U,V when needed.
- I moved `avifChannelOffset()` to `aviftest_helpers.h`. Note that I find "offset" less confusing because "avifChannelIndex" already exists as en enum.

`include/avif/internal.h` is currently inaccessible from tests.